new/LOC7373Atrib.xml

```diff
@@ -0,0 +1,71 @@
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-05-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place sameAs="wd:Q755734" type="town">
+                    <placeName xml:id="n1" xml:lang="en">Athribis</placeName>
+                    <placeName xml:id="n2" xml:lang="en">Athribis in Lower Egypt</placeName>
+                    <placeName xml:id="n3" xml:lang="gez">አትሪብ፡ </placeName>
+                    <placeName corresp="#n3" xml:lang="gez" type="normalized">ʾAtrib</placeName>
+                    <placeName xml:lang="grc" xml:id="n4">Ἄθλιβις</placeName>
+                    <placeName corresp="#n4" xml:lang="grc" type="normalized">Áthlibis</placeName>
+                    <placeName xml:lang="grc" xml:id="n5">Ἄθλιβις Κάτω Αιγύπτου</placeName>
+                    <placeName xml:lang="grc" corresp="#n5" type="normalized">Athlibis Kátō Aigyptū</placeName>
+                    <placeName xml:lang="grc" xml:id="n6">Ἄθριβις</placeName>
+                    <placeName xml:lang="grc" corresp="#n6" type="normalized">Athribis</placeName>
+                    <placeName xml:lang="ar" xml:id="n7">أتريبس</placeName>
+                    <placeName corresp="#n7" xml:lang="ar" type="normalized">ʾAtrībis</placeName>
+                    <placeName xml:id="n8" xml:lang="ar">أثريب</placeName>
+                    <placeName corresp="#n8" xml:lang="ar" type="normalized">ʾAṯrīb</placeName>
+                    <placeName xml:id="n9" xml:lang="cop">ⲁⲑⲣⲏⲃⲓ</placeName>
+                    <placeName corresp="#n9" type="normalized" xml:lang="cop">Athrēbi</placeName>
+                </place>
+                <listRelation>
+                    <relation name="saws:isAttributedToAuthor" active="LOC7373Atrib" passive="PRS7054Michael"/>
```

Comment:
you cannot attribute places to authors, only works
please read https://betamasaheft.eu/Guidelines/?q=saws&id=relations

Reply:
OK.